### ISSUE TITLE
Browser-based demo ideas?

### ISSUE THREAD

**Issue by paulkernfeld** [mappum/webcoin#15]
I'm interested in implemented a browser-based demo that shows off the power of webcoin. Any ideas about this?

One pretty simple idea is to show activity from a hard-coded list of [richest bitcoin addresses](http://www.bitcoinrichlist.com/top100).

It would also be cool to show a visualization of all transactions that are occurring, but that would require a lot of bandwidth from web seeds.

I think an issue that would need to be tackled is web seed reliability, which I mentioned in #5.

**Comment by mappum** [comment]
I think that's a good idea, maybe we should just start with a simple block explorer since the block header sync happens very quickly, and we can download full blocks when necessary on an ad-hoc basis. This wouldn't be able to look up addresses without a blockchain scan, but it could show newly announced transactions.

>that would require a lot of bandwidth from web seeds

Do you mean the web bridges? Because the seeds only handle addresses and signaling of peers, they don't deal with the actual data. Web bridges would use bandwidth relaying the transactions, but not any more than a normal TCP peer connection.

**Comment by paulkernfeld** [comment]
Yes, sorry, I meant web bridges, not web seeds. You're right that this will work fine once enough people are running bridge nodes, but if there are only a couple bridges, there might be a lot of load on them.

**Comment by paulkernfeld** [comment]
Yes, sorry, I meant web bridges, not web seeds. You're right that this will work fine once enough people are running bridge nodes, but if there are only a couple bridges, there might be a lot of load on them.

**Comment by mappum** [comment]
Right, we'll have to do our best to get people to run bridges. It may make
sense to run then by default for webcoin clients running on Node.js.

**Comment by latimesmike** [comment]
Hello, I'm new here and I'm not sure this is possible with WebCoin, but I was wondering if it was possible to create a simple checkout demo?

Something like monitoring an address for an incoming payment, and show a success message once the full price has been paid? I was having trouble getting an easy/low cost way to do this for another project, and services like Blockcypher charge thousands of dollars for full access to their API that monitors addresses. Might be a useful demo to show developers how easy it could be to process a payment with a the WebCoin library.

**Comment by Head** [comment]
Hi @latimesmike wouldn't be that hard, you can look at my code at [Gotoshi](https://github.com/Head/gotoshi/blob/master/src/app/services/bitcoinNode.js) what you need would be the constructor stuff and maybe reading the raw transactions, look at the function decodeTransaction (maybe not the best implementation, but it worked for my purpose)

**Comment by paulkernfeld** [comment]
I put up a simple demo, "Bitcoin Eater Live Feed," [here](http://paulkernfeld.com/burnie/). It just shows a live feed of bitcoins sent to the address 1BitcoinEaterAddressDontSendf59kuE. Hopefully this will be one of many demos!

**Comment by mappum** [comment]
@Head @paulkernfeld Nice webcoin applications! We should build a list of applications that people can demo and put it in the wiki or README.
